### PR TITLE
March 2024 OTP-UI Update

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Updates all OTP-UI packages. Uses yarn version override to get around uFuzzy issue.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Thank you!

File: yarn.lock [hunk 1]
```diff
@@ -2405,6 +2410,24 @@
     lodash.isequal "^4.5.0"
     qs "^6.9.1"
 
+"@opentripplanner/core-utils@^11.3.1":
+  version "11.3.1"
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2414]: Can this be deduplicated?

File: yarn.lock [hunk 2]
```diff
@@ -2434,6 +2457,17 @@
     isomorphic-mapzen-search "^1.6.1"
     lodash.memoize "^4.1.2"
 
+"@opentripplanner/geocoder@^2.2.0":
+  version "2.2.0"
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2461]: Same here

File: yarn.lock [hunk 3]
```diff
@@ -2465,10 +2499,28 @@
     react-resize-detector "^4.2.1"
     string-similarity "^4.0.4"
 
-"@opentripplanner/location-field@^2.0.15":
-  version "2.0.15"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/location-field/-/location-field-2.0.15.tgz#c13d59967d9335b3be96a02fb8f3b992a7814eac"
-  integrity sha512-Gg1Y9SLt3UO3+7KvwJc7gQWxSD03h8XSaDo5yz06IfMfqu6H1dBM1bZvFWuEWZwQwwpH9o7wrMw9ygraH2UMIg==
+"@opentripplanner/itinerary-body@^5.2.8":
+  version "5.2.8"
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2503]: Same here

**Review by binh-dam-ibigroup — COMMENTED** [review]
Can the OTP-UI packages be deduplicated?

**Review by binh-dam-ibigroup — APPROVED** [review]
Thanks for deduplicating

**Pull request merged**